Feature/datadog ecs (#131)
ExpediaGroup/apiary-federation: README.md

| tcp_keepalive_probes               | Sets net.ipv4.tcp_keepalive_probes (seconds), currently only supported in ECS.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    | number |      `2`       | no |

> | datadog_key_secret_name            | SecretsManager secret name containing a DataDog API key. This needs to be created manually in AWS secrets manager.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                | number |     `200`      | no |

Reviewer: ```suggestion
| datadog_key_secret_name            | Name of the secret containing the DataDog API key. This needs to be created manually in AWS secrets manager.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                | number |     `200`      | no |
```
